classes/migration/upgrade/v3_6_0/I10406_EditorialTasks.php

```diff
@@ -110,6 +117,7 @@ public function up(): void
             $table->foreign('email_template_id')
                 ->references('email_id')
                 ->on('email_templates');
+            $table->string('title', 255);
```

Comment:
You can remove this, the title is already added into the edit_task table. In the next migration I'll remove it from the notes table, as a title previously was recorded in the headnote